aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/Program.cs

```diff
@@ -0,0 +1,59 @@
+using Data;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+var connection = new SqliteConnection("DataSource=:memory:");
+connection.Open();
+
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.Services.AddDbContext<TodoGroupDbContext>(options =>
+{
+    options.UseSqlite(connection);
+});
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    // localhost:{port}/swagger
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.MapGet("/", () => "Hello World!");
+
+// todo endpoints
+var todos = app.MapGroup("/todos").WithTags("Todo Endpoints");
+todos.MapGet("/", RouteHandlers.GetAllTodos);
+todos.MapGet("/{id}", RouteHandlers.GetTodo);
+todos.MapPost("/", RouteHandlers.CreateTodo).AddRouteHandlerFilter((context, next) =>
+{
+    if (context.HttpContext.Request.ContentLength > 80)
```

Comment:
Let's think of something else a filter can do. Can we just have it log something about the request?

Assuming that `Request.ContentLength` is not null. or not realizing that a null `ContentLength` means the request body size is unbounded, is a common security issue. See https://github.com/dotnet/aspnetcore/issues/43080.